src/debugpy/server/adapters.py

```diff
@@ -346,6 +350,40 @@ def next_request(self, request: Request):
         self._tracer.step_over(thread)
         return {}
 
+    def gotoTargets_request(self, request: Request) -> dict:
+        path = request("source", dict)["path"]
+        line = request("line", int)
+        target_id = self._goto_targets_map.obtain_key((path, line))
+        target = {"id": target_id, "label": "%s:%s" % (path, line), "line": line}
```

Comment:
`%` is sort of legacy in modern Python. For cases where localization is not necessary, f-strings should be preferred: `f"{path}:{line}"`.

For display strings where placeholder order can vary when localized, the ideal form is `str.format()` with kwargs, i.e.: `"{path}:{line}".format(path=..., line=...)`. Existing code doesn't follow this, though, and mostly uses f-strings even then.

Reply:
Ah I just copied this from pydevd, I'll switch it to use a format string.